app/utils/fields-validations.ts

```diff
@@ -28,8 +32,8 @@ function validate(obj: object, schema: any, abortEarly: boolean = false): Valida
 }
 
 function textAreaValidation(text: string, theKey: string): ValidationErrors | null {
-  const schema = Joi.object({
-    [theKey]: Joi.string()
+  const schema = joi.object({
```

Comment:
I would probably only use the extended version of joi for those cases needed...so just for phone number